src/plugins/saved_objects_management/public/lib/get_saved_object_counts.ts

```diff
@@ -30,14 +30,18 @@
 
 import { HttpStart } from 'src/core/public';
 
+interface SavedObjectCountOptions {
```

Comment:
should this be exported?

Reply:
Yes. Updated this to be exported and used the new interface when creating options for the `getSavedObjectCounts` API.